modules/govuk/manifests/nodes/postgresql_db_admin.pp

```diff
@@ -64,16 +69,21 @@
   # We don't actually want to run a local PostgreSQL server, just
   # configure the RDS one
   class { '::postgresql::server':
+    package_ensure           => $ensure,
     default_connect_settings => $default_connect_settings,
     service_manage           => false,
   } ->
 
   service { 'postgresql':
-    ensure  => stopped,
+    ensure => stopped,
   }
 
-  include ::govuk_postgresql::server::not_slave
+  class { '::govuk_postgresql::server::not_slave':
+    ensure => $ensure,
+  }
```

Comment:
I think we might have to put this bit within a conditional of if ensure is present. I'm expecting that `service 'postgresql' ...` may error if that service doesn't exist and it looks like the not_slave only runs queries on a postgres service so an `ensure => absent` on it is a bit misleading

Reply:
Ah okay, good point. Have added a conditional here, instead of in the `not_slave` class.